Makefile

```diff
@@ -0,0 +1,167 @@
+.ONESHELL:
+SHELL := /bin/bash
+MODERNCVDIR =.
+MANUALDIR = $(MODERNCVDIR)/manual
+
+# version and date of the current release. This gets updated upon calling
+# either the rule version or the rule release
+VERSION = v2.1.0-40-gfe4d968-dirty
+VERSIONDATE = 2021/01/25
```

Comment:
Please don't add these hard-coded here. They shouldn't be hardcoded in a build file.

Reply:
They are not. Well, sorta not. 
When
```make version NEW=<ver_num>```
is  called both VERSION and VERSIONDATE get updated to the latest numbers. I.e., the Makefile itself gets updated, see lines 115 and 118.

Reply:
Yes, but I don't think they should be in here, especially if they get updated anyway. The versioning only happens once: if one wants to do a release, the version is taken from somewhere (I guess an env var is fine), the date is stored in a variable, and all files where that information is _necessary_ will be updated. Then the changes will be commited and tagged. At no other point version information is necessary, this is only for CTAN.